ethereum/core/src/main/java/tech/pegasys/teku/core/signatures/record/ValidatorSigningRecord.java

```diff
@@ -92,24 +139,22 @@ public Bytes toBytes() {
   }
 
   private boolean isSafeSourceEpoch(final UInt64 sourceEpoch) {
-    return attestationSourceEpoch.equals(NEVER_SIGNED)
-        || attestationSourceEpoch.compareTo(sourceEpoch) <= 0;
+    return attestationSourceEpoch == null || attestationSourceEpoch.compareTo(sourceEpoch) <= 0;
```

Comment:
We should be consistent with using `NEVER_SIGNED` rather than using `null` directly.